### PR TITLE
Bugfix/wcwpp 125/change to live url

### PR DESCRIPTION
**Pull request by Martin-Welte**
# TL;DR

- exchanged testing urls for live urls
- added composer script to generate installable package

### CODE REVIEW

**Review by JeanninePrueger — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -47,6 +47,22 @@
...
+        "rm -rf tests/ || true",
+        "rm .editorconfig || true",
+        "rm .gitignore || true",
+        "rm .phpcs.xml || true",
+        "rm package.json || true",
+        "rm package-lock.json || true",
+        "rm phpunit.xml.dist || true",
+        "rm README.md || true",
+        "rm commitlint.config.js || true",
+        "mv .env.example .env"
```
Comment (lucapircher) [inline composer.json]: I would not ship the package with a dotenv file. The default values are the values needed for production anyways.

File: .env.example [hunk 2]
```diff
@@ -1,2 +1,2 @@
-GEBRUEDER_WEISS_OAUTH_TOKEN_URL=https://oauth.gebrueder-weiss-woocommerce.towa-online.at/oauth/token
-GEBRUEDER_WEISS_API_URL=https://api.gebrueder-weiss-woocommerce.towa-online.at
+GEBRUEDER_WEISS_OAUTH_TOKEN_URL=https://my.api.gw-world.com/token
```
Comment (lucapircher) [inline .env.example]: I'd keep the mock services in the .env.example since these should be used for development.

**Review by lucapircher — COMMENTED** [review]
The API client should also be updated with the production URL. The test environment URL is specified in the API Schema and hence is used as a default value in the API client.

File: composer.json (hunk 1 above)
Comment (Martin-Welte) [reply composer.json]: is updated

File: .env.example (hunk 2 above)
Comment (Martin-Welte) [reply .env.example]: good point :)

**Review by mattcrn — APPROVED** [review]

**Review by JeanninePrueger — APPROVED** [review]

**Pull request merged**